src/test/java/hwr/oop/budgetbook/ReadWriteCsvTest.java

```diff
@@ -0,0 +1,46 @@
+package hwr.oop.budgetbook;
+
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+
+public class ReadWriteCsvTest {
+    @Test
+    void readCsvFile_TryIfATestFileIsReadAndDisplayed() {
+        String path = "C:\\Users\\Malte\\IdeaProjects\\hwr-oop-project-bank\\src\\test\\java\\hwr\\oop\\budgetBook\\test.csv";
```

Comment:
Lass uns herausfinden ob es auch möglich ist realative Pfade zu verwenden. Spätestens bei mir wird der Test nicht funktionieren. Das ist ja nicht der Sinn eines Tests.

Reply:
wenn man keinen path angeibt sondern nur den file name macht der es in dem folder in dem du gerade bist. ansonsten bietet intelij folgende optionen an path zu kopieren. bringt uns eine davon was 
![image](https://user-images.githubusercontent.com/103031906/167406104-13263f03-44b4-4a0a-abb2-a9fe422203a0.png)